### ISSUE TITLE
Github Action "Latest IJ" is failing with compilation error

### ISSUE THREAD

**Issue by adietish** [redhat-developer/intellij-quarkus#1219]
https://github.com/redhat-developer/intellij-quarkus/actions/runs/6450808350/job/17510609341

> Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.

> /home/runner/work/intellij-quarkus/intellij-quarkus/intellij-community/plugins/gradle/testSources/org/jetbrains/plugins/gradle/importing/TestGradleBuildScriptBuilder.kt: (155, 11): Unresolved reference: isSupportedJavaLibraryPlugin

**Comment by angelozerr** [comment]
@fbricon do you think we can close this issue?

**Comment by fbricon** [comment]
No: https://github.com/redhat-developer/intellij-quarkus/actions/workflows/IJ-latest.yml
